README.md

```diff
@@ -1,2 +1,7 @@
 # Gymkhana-IITMandi
 Website for Gymkhana - IIT Mandi
+
+for news updates :
```

Comment:
You can change this readme to markdown for displaying it better in future commits